src/api/api.controller.ts

```diff
@@ -129,12 +126,17 @@ export class ApiController {
 
   @Get('/dev-platform-data/:profileId')
   @IdCheck('profileId')
+  @UseGuards(AuthorizationGuard)
   @ApiOperation({ summary: "Get Developer's Platform Data" })
-  async getdpd(@Param('profileId') profileId: string): Promise<Ok<Api[]>> {
+  async getdpd(
+    @Param('profileId') profileId: string,
+    @Query('profileId') profile_id: string,
```

Comment:
Not sure u need the @Query u defined here...

Reply:
I do, I need to get the profile Id to be able to compare, though it may not be needed in the service, also the input field needs to appear in swagger ui

Reply:
oh that's true...